gtsfm/averaging/rotation/rotation_averaging_base.py

```diff
@@ -82,22 +82,22 @@ def create_computation_graph(
         self,
         num_images: int,
         i2Ri1_graph: Delayed,
-        i2Ti1_priors: Dict[Tuple[int, int], PosePrior],
+        i1Ti2_priors: Dict[Tuple[int, int], PosePrior],
```

Comment:
was it just for the priors that we used i1Ti2 or for the rotations too? 
I wasn't sure if it was already i1Ti2 on master, or was that change part of the Hilti PR.

Reply:
We have to change it as part of Hilti